apps/docusaurus/docs/tools/aptos-cli/use-cli/working-with-move-contracts.md

```diff
@@ -4,7 +4,7 @@ title: "Working With Move Contracts"
 
 # Working With Move Contracts
 
-The Aptos CLI is mostly used to compile, test, and formally verify Move contracts. If you have not installed the Aptos CLI yet, you can do so by following the steps here [Install the Aptos CLI (In-between Page)](https://www.notion.so/Install-the-Aptos-CLI-In-between-Page-9b923ca3507c4ffd8291f905ba621b95?pvs=21).
+The Aptos CLI is mostly used to compile, test, and formally verify Move contracts. If you have not installed the Aptos CLI yet, you can do so by following the steps here [Install the Aptos CLI](../install-cli/index.md).
```

Comment:
Did you mean to edit this?

Reply:
Yeah, that's a mistake from earlier - that new relative link is correct.